scripts/upload-zynq7000-smt2.gdb

```diff
@@ -0,0 +1,16 @@
+target remote localhost:3333
```

Comment:
maybe we could add single-line comment regarding the setup (eg. openocd gdb server with onboard zynq7000 Zedboard Digilent FTDI-based SMT2 JTAG)